ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollViewManager.java

```diff
@@ -47,6 +47,11 @@ public void setShowsVerticalScrollIndicator(ReactScrollView view, boolean value)
     view.setVerticalScrollBarEnabled(value);
   }
 
+  @ReactProp(name = "scrollEnabled")
+  public void setScrollEnabled(ReactScrollView view, boolean value) {
```

Comment:
I was wondering where would I put that, thanks, I have added this to the latest revision.